src/ripple/nodestore/impl/DatabaseNodeImp.cpp

```diff
@@ -70,13 +70,8 @@ DatabaseNodeImp::fetchNodeObject(
         switch (status)
         {
             case ok:
-                ++fetchHitCount_;
-                if (nodeObject)
-                {
-                    fetchSz_ += nodeObject->getData().size();
-                    if (cache_)
-                        cache_->canonicalize_replace_client(hash, nodeObject);
-                }
+                if (nodeObject && cache_)
+                    cache_->canonicalize_replace_client(hash, nodeObject);
```

Comment:
We lost additions to both `fetchHitCount_` and `fetchSz_` here.  I'm fine with that as long as that's really what you want and not an unintended side effect.